data/locale/ca-ES.ini

```diff
@@ -2,7 +2,6 @@ NDIPlugin.Default="Per defecte"
 NDIPlugin.NDISourceName="Orígen NDI™"
 NDIPlugin.SourceProps.SourceName="Nom de l'origen"
 NDIPlugin.SourceProps.Bandwidth="Ample de banda"
-NDIPlugin.SourceProps.Sync="Sincronització"
```

Comment:
Why get rid of `NDIPlugin.SourceProps.Sync`?
It is still used in `src\obs-ndi-source.cpp`

Reply:
You are correct, though the English string was updated with https://github.com/obs-ndi/obs-ndi/commit/46fd699538ebcfdadabc5d63782d5ab32cc8bd92#diff-b3c242ee2d502c9e6dade4bace60d2ea2ab52969dd96b1cac4ed42f60668b18aR5 and later again with https://github.com/obs-ndi/obs-ndi/commit/a07ad386b240d24adc8e57b7aef5de921f020d88#diff-b3c242ee2d502c9e6dade4bace60d2ea2ab52969dd96b1cac4ed42f60668b18aR5, but not the Catalan version.
I rather remove a wrong translation and therefore use the English variant.